test/test_core.jl

```diff
@@ -867,7 +867,7 @@ end
                     else
                         @NT y = x.y + y
                     end), t, :x, select=:z) == table([1, 2], [6, 15], names=Symbol[:x, :y])
-    @test groupreduce(:y => (+), t, :x, select=:z) == table([1, 2], [6, 15], names=Symbol[:x, :y])
+    @test groupreduce((:y => (+),), t, :x, select=:z) == table([1, 2], [6, 15], names=Symbol[:x, :y])
```

Comment:
I think it'd be nice to keep this, it's just a convenience thing. `groupby` should also support this...

Reply:
Ok, then I'll implement it properly for both, here it worked by chance (for example, it wouldn't have worked with a `NDSparse`).

Reply:
Fixed.